dev/reference/tools/download.md

```diff
@@ -35,6 +35,18 @@ Download [TiDB Lightning](/dev/reference/tools/tidb-lightning/overview.md) by us
 >
 > `{version}` in the above download link indicates the version number of TiDB Lightning. For example, the download link for `v3.0.5` is `http://download.pingcap.org/tidb-toolkit-v3.0.5-linux-amd64.tar.gz`. You can also download the latest unpublished version by replacing `{version}` with `latest`.
 
+## BR (backup and restore)
+
+Download [BR](/dev/how-to/maintain/backup-and-restore/br.md) by using the download link in the following table:
```

Comment:
What about `If you want to download the latest version of BR, directly download the TiDB package, because BR is included in the TiDB package.`?